src/functions/cr.js

```diff
@@ -0,0 +1,57 @@
+//@flow
+// Row breaks within tabular environments, and line breaks at top level
+
+import defineFunction from "../defineFunction";
+import buildCommon from "../buildCommon";
+import mathMLTree from "../mathMLTree";
+import { calculateSize } from "../units";
+import ParseError from "../ParseError";
+
+defineFunction({
+    type: "cr",
+    names: ["\\\\", "\\cr", "\\newline"],
+    props: {
+        numArgs: 0,
+        numOptionalArgs: 1,
+        argTypes: ["size"],
+        allowedInText: true,
+    },
+
+    handler: (context, args, optArgs) => {
+        return {
+            type: "cr",
+            // \\ and \cr both end the row in a tabular environment
+            // This flag isn't currently needed by environments/array.js
+            //newRow: context.funcName !== "\\newline",
+            // \\ and \newline both end the line in an inline math environment
+            newLine: context.funcName !== "\\cr",
```

Comment:
I think we can do something like:
```
            newLine: context.funcName !== "\\cr" && (!context.parser.settings.displayMode || nonStrict),
```

Reply:
In LaTeX, `\\` "works" (does nothing) in display mode, same as the current code.  Your change would cause an error in strict mode.  But that's true, we could drop the "cr" class in strict mode; good idea.